geode-core/src/main/java/org/apache/geode/management/GatewaySenderMXBean.java

```diff
@@ -273,6 +273,8 @@
    */
   int getEventsExceedingAlertThreshold();
 
-
-
+  /**
```

Comment:
you don't need these methods in the mbeans anymore

Reply:
true